creusot/src/translation/function/statement.rs

```diff
@@ -169,6 +170,7 @@ impl<'tcx> BodyTranslator<'_, 'tcx> {
                 Operand::Constant(crate::constant::from_ty_const(
                     self.ctx,
                     *len,
+                    Ty::new_usize(self.ctx.tcx),
```

Comment:
strictly speaking we should be able to do `self.ctx.tcx.types.usize` here, but not important